front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.html

```diff
@@ -144,10 +144,10 @@ <h3>Covering period</h3>
   </div>
```

Comment:
There are 4 issues flagged by ANDI
![app-138-andi-errors](https://user-images.githubusercontent.com/2971785/166690623-c58a346a-24a5-4489-9875-f87f022f7563.png)

Reply:
This seems to be an issue with primeng's selectButton
<img width="1156" alt="image" src="https://user-images.githubusercontent.com/97105825/166716565-d32bd758-fd84-4cd3-bc78-016aa55d0b1a.png">